src/webgpu/gpu_test.ts

```diff
@@ -67,6 +67,17 @@ import {
 import { createTextureFromTexelViews } from './util/texture.js';
 import { reifyExtent3D, reifyOrigin3D } from './util/unions.js';
 
+// Declarations for WebGPU items we want tests for that are not yet officially part of the spec.
+declare global {
+  // MAINTENANCE_TODO: remove once added to @webgpu/types
+  interface GPUSupportedLimits {
+    readonly maxStorageBuffersInFragmentStage: number;
+    readonly maxStorageTexturesInFragmentStage: number;
+    readonly maxStorageBuffersInVertexStage: number;
+    readonly maxStorageTexturesInVertexStage: number;
```

Comment:
These attributes should probably be optional (`?`) for now to have typescript help us avoid assuming they exist. We can always use `!` to avoid unnecessary checks (like when we're already in `if (t.isCompatibility)`)

Reply:
done